.circleci/common.py

```diff
@@ -36,6 +36,7 @@ def get_airflow_version(ac_version):
     ("2.1.4-2", ["buster"]),
     ("2.2.0-4.dev", ["bullseye", "buster"]),
     ("2.2.1-2.dev", ["bullseye", "buster"]),
+    ("2.2.2-1.dev", ["bullseye", "buster"]),
```

Comment:
```suggestion
    ("2.2.2-1.dev", ["bullseye"]),
```

We can skip buster for `2.2.2-1` completely and hopefully Users have got a chance to known about images with no "buster" tag name

Reply:
Done. I wasn't sure if we were fully confident nothing in the stack still needed it.